src/datepicker/datepicker.js

```diff
@@ -64,7 +54,7 @@ export default class Datepicker extends React.Component<
       isPseudoFocused,
       ...(calendarFocused === null ? {} : {calendarFocused}),
     });
-    this.props.onChange(data);
+    this.props.onChange && this.props.onChange(data);
```

Comment:
Why not keep the default prop so we don't need to make one liner checks every time we call the handler?

Reply:
the `onChange` prop is optional in types now, so we need to check at each call-site